torch/_inductor/compile_fx.py

```diff
@@ -1300,7 +1300,7 @@ def compile_fx(
         with config.patch(
             {
                 "cpp_wrapper": False,
-                "triton.autotune_at_compile_time": True,
+                "triton.autotune_at_compile_time": config.triton.autotune_at_compile_time,
```

Comment:
Can you also update here, https://github.com/pytorch/pytorch/blob/main/torch/_inductor/config.py#L700, to make the default value to 1? I wanted to keep the always on unless it breaks anything beyond the current test coverage. Not sure if your change to wrapper.py fixes the underlying problem. If yes, you can make the default value of `TORCHINDUCTOR_TRITON_AUTOTUNE_AT_COMPILE_TIME` always to 1; otherwise, you can make the default of `TORCHINDUCTOR_TRITON_AUTOTUNE_AT_COMPILE_TIME` different between fbcode and OSS.

Reply:
Making fbcode and OSS default difference since the issue still exists